file-formats/edot/type/zif_aff_edot_v1.intf.abap

```diff
@@ -0,0 +1,101 @@
+"! <p class="shorttext synchronized">AFF for eDocument Type (EDOT)</p>
+"! eDocument types that can be used in eInvoice process
+INTERFACE zif_aff_edot_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Read Access Loggging Settings</p>
+    "! Read access logging settings
+    BEGIN OF ty_read_access_logging_setting,
+      "! <p class="shorttext">File Type</p>
+      "! File type
+      "! $required
+      file_type        TYPE c LENGTH 10,
+      "! <p class="shorttext">File Structure Type</p>
+      "! File structure type
+      "! $required
+      file_structure   TYPE c LENGTH 30,
+      "! <p class="shorttext">File Description</p>
+      "! File description
+      file_description TYPE c LENGTH 60,
+    END OF ty_read_access_logging_setting.
+  "! <p class="shorttext">Read Access Logging Settings</p>
+  "! Read access logging settings
+  TYPES ty_read_acc_logging_settings TYPE SORTED TABLE OF ty_read_access_logging_setting WITH UNIQUE KEY file_type.
+  TYPES:
+    "! <p class="shorttext">eDocument Type Specific Additional Tables</p>
+    "! eDocument type specific additional tables
+    BEGIN OF ty_additional_table,
+      "! <p class="shorttext">Sequence Number</p>
+      "! Sequence number
+      "! $required
+      "! $minimum 1
+      "! $maximum 9999
+      sequence_number TYPE i,
+      "! <p class="shorttext">Table Name</p>
+      "! Table name
+      "! $required
+      table_name      TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_additional_table.
+  "! <p class="shorttext">Read Access Logging Settings</p>
+  "! Read access logging settings
+  TYPES ty_additional_tables TYPE SORTED TABLE OF ty_additional_table WITH UNIQUE KEY sequence_number.
+  TYPES:
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">eDocument Table Name</p>
+      "! eDocument table name
+      table_name                     TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Preprocess Before Archiving</p>
+      "! Preprocess before archiving
+      is_archive_preprocess_needed   TYPE abap_bool,
+      "! <p class="shorttext">Created in Contingency Mode</p>
+      "! Created in contingency mode
+      is_created_in_contingency_mode TYPE abap_bool,
```

Comment:
Would you align the field names and the title's please?

Furthermore, is any field of `general_information` a `$required` field?